[2025-09-26T17:08:19.182+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-26T17:08:19.260+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_verify.task_3 scheduled__2025-09-25T00:00:00+00:00 [queued]>
[2025-09-26T17:08:19.285+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_verify.task_3 scheduled__2025-09-25T00:00:00+00:00 [queued]>
[2025-09-26T17:08:19.287+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-09-26T17:08:19.329+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): task_3> on 2025-09-25 00:00:00+00:00
[2025-09-26T17:08:19.398+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=224) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-26T17:08:19.383+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_verify', 'task_3', 'scheduled__2025-09-25T00:00:00+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/dag_verify.py', '--cfg-path', '/tmp/tmp2ocscjoi']
[2025-09-26T17:08:19.410+0000] {standard_task_runner.py:72} INFO - Started process 241 to run task
[2025-09-26T17:08:19.410+0000] {standard_task_runner.py:105} INFO - Job 177: Subtask task_3
[2025-09-26T17:08:19.945+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_verify.task_3 scheduled__2025-09-25T00:00:00+00:00 [running]> on host 4461bcf72552
[2025-09-26T17:08:20.305+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_verify' AIRFLOW_CTX_TASK_ID='task_3' AIRFLOW_CTX_EXECUTION_DATE='2025-09-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-25T00:00:00+00:00'
[2025-09-26T17:08:20.309+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-26T17:08:21.104+0000] {logging_mixin.py:190} INFO - Matplotlib version: 3.9.2
[2025-09-26T17:08:21.105+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-09-26T17:08:21.124+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-26T17:08:21.125+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_verify, task_id=task_3, run_id=scheduled__2025-09-25T00:00:00+00:00, execution_date=20250925T000000, start_date=20250926T170819, end_date=20250926T170821
[2025-09-26T17:08:21.202+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-26T17:08:21.241+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-26T17:08:21.244+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
